dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdNdkSymbolFileUploadTask.kt

```diff
@@ -70,12 +71,23 @@ internal abstract class DdNdkSymbolFileUploadTask @Inject constructor(
         }
     }
 
+    // Map of Android architecture names to the architecture names recognized by the symbolication service
+    data class SupportedArchitectureMapping(
+        val arch: String,
+        val uploadArch: String
+    )
+
     companion object {
         internal const val TASK_NAME = "uploadNdkSymbolFiles"
         internal const val KEY_NDK_SYMBOL_FILE = "ndk_symbol_file"
         internal const val TYPE_NDK_SYMBOL_FILE = "ndk_symbol_file"
         internal const val ENCODING = "application/octet-stream"
-        internal val SUPPORTED_ARCHS = setOf("armeabi-v7a", "arm64-v8a", "x86", "x86_64")
+        internal val SUPPORTED_ARCHS = setOf(
+            SupportedArchitectureMapping("armeabi-v7a", "arm"),
+            SupportedArchitectureMapping("arm64-v8a", "arm64"),
+            SupportedArchitectureMapping("x86", "x86"),
+            SupportedArchitectureMapping("x86_64", "x64")
+        )
```

Comment:
Sounds like could be simply `mapOf<String, String>`, without introducing data class. But not a big deal.

Reply:
Yeah, I was curious about that. I could go either way but I figured the data class was more descriptive.

Reply:
just lookup is now linear instead of O(1), but it doesn't really matter here